binder/getting-started/getting_started.md

```diff
@@ -0,0 +1,68 @@
+# Elyra quickstart
+
+This short interactive quickstart provides an introduction to Elyra - an AI centric extension for JupyterLab. Note that this tour does not include features requiring Software that is not bundled with Elyra or JupyterLab, such as the Notebook pipeline editor, which requires a  KubeFlow Pipelines installation. 
```

Comment:
nit: should `Software` be capitalized?, extra space before `Kubeflow`(doesn't affect rendering)

Reply:
fixed in follow-up PR https://github.com/elyra-ai/examples